testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -109,11 +110,19 @@ type OwnerStakeDelegation {
   pools: [Pool!]!
 }
 
+"""Information about pagination in a connection"""
 type PageInfo {
-  hasNextPage: Boolean!
+  """When paginating backwards, are there more items?"""
```

Comment:
Yay, docs! This is a test schema, or can this also be normative and used as a doc?

Reply:
Well, docs were always a thing, even with juniper. This file is actually generated with the introspection system (I mean, we launch a jormungandr instance with the explorer enabled, and ask it for the schema), so it's no different than the schema that's actually served, and this docs can be seen from the playground, for example